styles/hud/cgaz.scss

```diff
@@ -18,3 +19,54 @@
 		transform: rotatez(180deg);
 	}
 }
+
+.line {
+	width: 100px;
+	margin-top: -1px;
+	border-top: 2px solid;
+	horizontal-align: center;
+	vertical-align: bottom;
+	overflow: noclip noclip;
+}
+
+.tick {
+	border-left: 2px solid black;
+	border-right: 2px solid black;
+	//margin-left set in js for positioning
+	margin-right: -1px;
+	vertical-align: top;
+	overflow: noclip noclip;
+
+	&__full {
+		height: 6px;
+	}
+
+	&__half {
+		height: 3px;
+	}
+}
+
+.stats {
+	width: 50px;
+	height: 24px;
+	font-size: 20px;
+	text-align: right;
+
+	&__container {
+		width: 200px;
+		height: 100px;
+		horizontal-align: center;
+		vertical-align: center;
+		overflow: noclip;
+	}
+
+	&__pitch {
+		horizontal-align: left;
+		margin-left: -40px;
```

Comment:
Negative margins are hacky and I try to only use them in cases with flow layout where they can be really useful. In this case `transform: translatex(-40px);` should work fine.

Reply:
yea, good look. I was in margin mode from the other panels in this change. Agreed, updated to use translate